woocommerce/woocommerce
# Add PR number to Jetpack changelogger entries

### All Submissions:






### Changes proposed in this Pull Request:

When a Jetpack Changelogger entry is made, Matticbot will append the pull request number as a link unless it already exists.


### How to test the changes in this Pull Request:

1. Create a new branch from this one locally
2. Add a changelog entry to WooCommerce without adding a pull request number (because you haven't create a PR yet)

```
cd plugins/woocommerce
./vendor/bin/changelogger add
```
3. Commit the changelog and push to Github
4. Create a test PR
5. Check the logs on the Changelog Github Action and see that Matticbot will append the changelog entry
6. Confirm the Action runs again, this time passing.

### Other information:




### Changelog entry

Add PR number to Jetpack changelogger entries

### FOR PR REVIEWER ONLY:
